### DISCUSSION TITLE
In Meow’s Keypad state, how to promote C-x instead of C-c?

### DISCUSSION THREAD

**Discussion by thedpws** [meow-edit/meow#591] [Q&A]
Hi! Any comments or questions are welcome.

At the moment, keypad performs `C-c` commands with no explicit `c` keypress, while performing `C-x` commands requires an explicit `x` keypress. 

I wish to reverse this behavior so that `C-c` commands require an explicit `c` keypress while `C-x` commands do not require an explicit `x` keypress.

**I’d like to gather some ideas on how to approach this!** Also, if someone can help me understand the design decision to promote C-c over C-x, I am very interested in knowing.

**Comment by DogLooksGood** [comment]
Please check the variable `meow-keypad-leader-dispatch`.

I think this is what you want
```
(setq meow-keypad-leader-dispatch "C-x")
```
